javascriptv3/example_code/dynamodb/src/movies/deleteItem.js

```diff
@@ -0,0 +1,43 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with the AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3.
+
+Purpose:
+deleteItem.js demonstrates how to use the Amazon DynamoDB document client to delete an item from an Amazon DynamoDB table.
+
+Inputs (replace in code):
+- TABLE_NAME
+- primaryKey - The name of the primary key. For example, "title".
+- VALUE_1: Value for the primary key (The format for the datatype must match the schema. For example,
```

Comment:
Add ending punctuation after "Value for the primary key."

Reply:
done